packages/pwa-kit-create-app/package.json

```diff
@@ -22,7 +22,7 @@
     "format": "internal-lib-build format \"**/*.{js,jsx}\"",
     "lint": "npm run lint:js",
     "lint:fix": "npm run lint:js -- --fix",
-    "lint:js": "internal-lib-build lint .",
+    "lint:js": "cross-env ../../node_modules/.bin/eslint \"**/*.{js,jsx}\"",
```

Comment:
Just curious: in this case, for what purpose is `cross-env` being used for?

Reply:
In this case cross-env will resolve the forward-slashes to it makes it work on windows machines.